### DISCUSSION TITLE
When uploading new media files, let me select a subfolder

### DISCUSSION THREAD

**Discussion by martinptacek** [directus/directus#13454] [Feature Requests]
Use case: Each record represents one estate property. Each property has list of pictures. It is practical to keep them in separated subfolders. It would be handy to let me chose subfolder when I upload a new file or make the upload path dynamic based e.g. on ID of the record.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
